test/local/user_agent.js

```diff
@@ -559,6 +559,36 @@ describe('userAgent', () => {
   )
 
   it(
+    'recognises new Firefox-iOS user agents',
+    () => {
+      parserResult = {
+        ua: {
+          family: 'Other'
+        },
+        os: {
+          family: 'Other'
+        },
+        device: {
+          family: 'Other'
+        }
+      }
+      const context = {}
+      const userAgentString = 'Firefox-iOS-FxA/6.0b42 (iPhone 6S; iPhone OS 10.3) (Firefox)'
+      const result = userAgent.call(context, userAgentString, log)
+
+      assert.equal(result.uaBrowser, 'Firefox')
+      assert.equal(result.uaBrowserVersion, '6.0')
+      assert.equal(result.uaOS, 'iOS')
```

Comment:
> So we'll have to work round it in our metrics pipeline lua scripts unfortunately.

I always make this mistake. Of course, we could also just update them with SQL and leave lua/re-importing stuff out of it.